blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx

```diff
@@ -59,41 +59,41 @@ const AuthorImage = ({ image, altText = "" }) => {
 	return null;
 };
 
-const Presentation = ({ author = {}, authorProfileLink }) => {
+const Presentation = ({ author }) => {
 	const phrases = usePhrases();
+
+	const originalAuthor = author?.additional_properties?.original;
```

Comment:
The field `additional_properties.original` does not exist on author data returned from the Author API. So this breaks the block (other than the image) on the author info templates:
![Screenshot 2024-06-06 at 3 14 08 PM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/d1c32e10-a7dc-4b32-b030-c8148b77434a)

You can test this by setting the global content in PB Editor to use the `author-api` and a slug of `tariq-alshwaiki`.

Reply:
It would have never worked that way but I will make some changes to allow it to work that way.